### PR TITLE
Website with navigation

### PR DESCRIPTION
**Pull request by amklass11**
@amklass11  and @Fatima-hub333  implemented
1.ui for the library app
2.linter errors fixed

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,25 +9,62 @@
...
+    <input class="w-50 p-1" type="text" id="titulo" placeholder="Title of Book">
+  </div>
+  <div class="mt-3 d-flex justify-content-center">
+    <label for="autor"></label>
+    <input  class="w-50 p-1" type="text" id="autor" placeholder="Author of Book">
+  </div>
+  <div class="d-flex justify-content-center">
+  <button id="btn">Add</button>
+</div>
+</form>
```
Comment (hiromataba) [inline index.html]: - [x] Please make your form match the given [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/books/images/awesome_books_full_website.png) now it looks like this :point_down: 

image

- Original design :point_down: 

image

File: index.html [hunk 2]
```diff
@@ -9,25 +9,62 @@
...
+    <h2 class="logo ms-3">Awesome Books</h2>
+    <ul class="d-flex gap-3">
+      <li><a id="booksBtn" href="#">List</a></li>
+      <li><a id="addbookBtn" href="#">Add NewBook</a></li>
+      <li><a id="contactBtn" href="#">Contact</a></li>
+    </ul>
+  </nav>
+</header>
+
+<main>
```
Comment (hiromataba) [inline index.html:22]: - [x] When any of the links are clicked, the main area changes to display only one of the three content sections but for now your main area is showing all three sections, you can add and remove CSS classes to display or hide the content by using the [Element.classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#examples) `add` and `remove` methods

image

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi  @amklass11  and @Fatima-hub333  :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lordkaito — APPROVED** [review]
Hi @amklass11, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**